### PR TITLE
added basic date validator and greaterThanNow rule

### PR DESCRIPTION
**Pull request by SimonSchick**
Woo, jojen!

### CODE REVIEW

**Comment by SimonSchick** [comment]
[Rotate home page featured streams](https://trello.com/c/DwlQbF6U/9-rotate-home-page-featured-streams)

File: lib/rules/date.js [hunk 1]
```diff
@@ -0,0 +1,33 @@
+import SyncRule from '../types/syncRule';
+
+// eslint-disable-next-line
+const strictISO = /^(\d{4}-[01]\d-[0-3]\dT[0-2]\d:[0-5]\d:[0-5]\d\.\d+([+-][0-2]\d:[0-5]\d|Z))|(\d{4}-[01]\d-[0-3]\dT[0-2]\d:[0-5]\d:[0-5]\d([+-][0-2]\d:[0-5]\d|Z))|(\d{4}-[01]\d-[0-3]\dT[0-2]\d:[0-5]\d([+-][0-2]\d:[0-5]\d|Z))$/;
+
+class DateValidator extends SyncRule {
+    coerce(value) {
+        if (typeof value === 'string' && strictISO.test(value)) {
```
Comment (connor4312) [inline lib/rules/date.js]: May want iso in a separate rule so that consumers can use multiple date formats

Comment (SimonSchick) [inline lib/rules/date.js]: We don't really support any other date format, do we?

Comment (connor4312) [inline lib/rules/date.js]: Not in our API, but it's totally feasible that we (or someone else, this is OSS) might want to use another format or consume one from a source we don't control.

**Pull request merged**